framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -220,11 +237,35 @@ private static TestCaseModeler getTestCaseModeler(String testDefFile)
 	    ObjectMapper objectMapper = new ObjectMapper();
 	    return objectMapper.readValue(new String(jsonData), TestCaseModeler.class);
 	  }
+
+  private static String getBaseName(String queryFile){
+  
+	    String extn = "";
+	    int t = 0;
+  	    while(queryFile.contains(".")){
+ 		t++;
+		extn = FilenameUtils.getExtension(queryFile);
+		if(!extn.contains("/")){
+	    		queryFile = FilenameUtils.removeExtension(queryFile);
+		}
+		else{
+			break;
+		}
+		if(t>4){
+			break;
+		}
+            }
+	    return queryFile;
+
+  }
+
   
   private static String getExpectedFile(String queryFile, String queryFileExt,
 	      String expectedFileExt) {
-	    int idx = queryFile.indexOf(queryFileExt.substring(2));
-	    return queryFile.substring(0, idx).concat(expectedFileExt.substring(2));
+	    
```

Comment:
Changed the getExpectedFile function as it was not supporting the test files that didnt have a .q or .sql in its filename .